package.json

```diff
@@ -18,8 +18,8 @@
     },
     "dependencies": {
         "@octokit/rest": "^15.9.4",
-        "@pulumi/awsx": "^0.18.0",
-        "@pulumi/pulumi": "^1.0.0-rc",
-        "@pulumi/random": "^1.0.0-rc"
+        "@pulumi/awsx": "0.18.14",
```

Comment:
@ellismg removing the caret from the version spec was an unintended change. Thanks for catching it. I have restored it now.

Reply:
> I have restored it now.

Thanks.  I think the lock file also needs to be updated with this change (since it encoded the requested version range inside the lock file itself).  Can you check if `yarn install` after these changes causes changes to the `yarn.lock`?

Reply:
Ah yes. Indeed. Let me regenerate the lock file.

Reply:
Done.